### PR TITLE
Carousel and Navbar fixed

### PR DESCRIPTION
**Pull request by peter-santana**
Navbar is fixed with routes and carousel is up and working (formatting to make it prettier is tbd)

### CODE REVIEW

File: frontend/src/components/Home/Home.js [hunk 1]
```diff
@@ -39,72 +36,76 @@ class Home extends Component {
...
+                                    <Carousel.Caption>
+                                        <h3>News</h3>
+                                    </Carousel.Caption>
+                                </Carousel.Item>
+
+                                <Carousel.Item>
+                                    <img
+                                        className="d-block w-100"
+                                        src={economicIndicators}
+                                        alt="Third slide"
```
Comment (crobles3) [inline frontend/src/components/Home/Home.js]: Change "Third Slide" to "Fourth Slide", just an observation, no big deal.

File: frontend/src/components/Home/Home.js [hunk 2]
```diff
@@ -39,72 +36,76 @@ class Home extends Component {
...
-                alt="Third slide"
-              />
-            </MDBView>
-          </MDBCarouselItem>
-        </MDBCarouselInner>
-      </MDBCarousel>
-    </MDBContainer> */}
+            <div>
+                <div className='container-fluid' >
+                    <div className="row">
```
Comment (crobles3) [inline frontend/src/components/Home/Home.js:41]: What's the purpose of this div with an empty h2?

File: frontend/src/components/Home/Home.js [hunk 3]
```diff
@@ -5,24 +5,21 @@ import { Nav, Navbar, NavDropdown, Card, Button, Carousel } from 'react-bootstra
...
-    render(){
+function Home() {
+
         return (
             <>
             <Navbar className="color-nav" >
                 
               <Container>
-              <Navbar.Brand href="#home">
+              <Navbar.Brand href="/">
```
Comment (crobles3) [inline frontend/src/components/Home/Home.js:22]: Either change the width or the height of the navbar brand to auto cause the logo looks a little stretched out. I would recommend letting the width as it is and setting the height to auto.

**Review by crobles3 — CHANGES_REQUESTED** [review]
can the Navbar be generalized into one component?

**Comment by peter-santana** [comment]
Comments by Christian have been fixed

**Review by crobles3 — APPROVED** [review]

**Pull request merged**